.github/workflows/consumer-contract-tests.yml

```diff
@@ -0,0 +1,103 @@
+name: Consumer contract tests
+# The purpose of this workflow is to validate the service level contract
+# using the Pact framework.
+#
+# More details on Contract Testing can be found in our handbook
+#
+# https://broadworkbench.atlassian.net/wiki/spaces/IRT/pages/2660368406/Getting+Started+with+Pact+Contract+Testing
+#
+#
+#
+# NOTE: The publish-contracts workflow will use the latest commit of the branch that triggers this workflow to publish the unique consumer contract version to Pact Broker.
+
+on:
+  pull_request:
```

Comment:
Shouldn't we only be publishing consumer contracts after merge, not on PR? @mspector any thoughts here? Comparing to Terra-UI's workflow for reference: https://github.com/DataBiosphere/terra-ui/blob/dev/.github/workflows/publish-pacts.yaml

Reply:
ah, yes, good catch. @dillydally414 is absolutely right